MockKitUI/View Controllers/MockCGMManagerSettingsViewController.swift

```diff
@@ -468,9 +521,36 @@ extension MockCGMManagerSettingsViewController: PercentageTextFieldTableViewCont
                 cgmManager.dataSource.effects.randomErrorChance = chance.clamped(to: 0...100)
             }
             tableView.reloadRows(at: [indexPath], with: .automatic)
+        case [Section.progress.rawValue, ProgressRow.percentComplete.rawValue]:
+            if let percentComplete = controller.percentage.map({ $0.clamped(to: 0...1) }) {
+                if cgmManager.mockSensorState.cgmStatusProgress == nil {
```

Comment:
[nit] Consider the following for these three blocks as it gets rid of the duplicate `.percentage.map {...}`.
```
if cgmManager.mockSensorState.cgmStatusProgress == nil {
    cgmManager.mockSensorState.cgmStatusProgress = MockCGMStatusProgress()
}
cgmManager.mockSensorState.cgmStatusProgress?.percentComplete = percentComplete
```